### ISSUE TITLE
Refinery 1.6.0 Docker image fails to start without arguments

### ISSUE THREAD

**Issue by martinmcnulty** [honeycombio/refinery#350]
**Versions**

- Refinery: 1.6.0 (Docker image)

**Steps to reproduce**

1. Create `refinery.toml` and `rules.toml` in a subdir called `conf`
2. Run `docker run -v $(pwd)/conf:/etc/refinery honeycombio/refinery:1.6.0`
3. The container exits immediately with an error message:

```
$ ls conf/
refinery.toml  rules.toml
$ docker run -v $(pwd)/conf:/etc/refinery honeycombio/refinery:1.6.0
command line parsing error - call with --help for usage
$
```

**Additional context**

In 1.5.2, the container started with no arguments:

```
$ docker run -v $(pwd)/conf:/etc/refinery honeycombio/refinery:1.5.2
time="2021-11-10T15:45:45Z" level=info msg="using specific identifier from config" identifier=192.168.1.1
time="2021-11-10T15:45:46Z" level=info msg="Peer list has changed. New peer list: [http://10.40.192.136:28957 http://10.40.192.139:22052 http://192.168.1.1:12346]"
...
```

**Comment by JamieDanielson** [comment]
Thank you for reporting @martinmcnulty! We'll get this fixed up.

**Comment by MikeGoldsmith** [comment]
Hey @martinmcnulty - we released [v1.6.1](https://github.com/honeycombio/refinery/releases/tag/v1.6.0) with a fix. Let us know if you have any more issues.

**Comment by martinmcnulty** [comment]
Looks good! Thanks!
